tests/test_check_stop_condition.py

```diff
@@ -47,14 +48,16 @@
 
     # Loop mimics path_2024.py structure
     start_time = time.time()
+    time_limit_evaluator = check_time_condition.CheckTimeCondition(start_time,MAXIMUM_FLIGHT_TIME)
```

Comment:
Nit: Add a space before `MAXIMUM_FLIGHT_TIME`

Reply:
Also, can we call the variable `check_time_condition`. In my opinion, it makes it more clear its an object of that class.